### PR TITLE
OSDOCS#5056: GA CgroupV2 in Openshift 4.13

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OSDOCS-5056

enterprise-4.13+

Installing -> Installation configuration -> [Enabling Linux control group version 2](https://56436--docspreview.netlify.app/openshift-enterprise/latest/installing/install_config/enabling-cgroup-v2.html)  Removed TP snippet, removed Step 2 to add feature gate 

Nodes -> Working with clusters -> [Configuring the Linux cgroup on your nodes](https://56436--docspreview.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-cgroups-2.html) Changed title from _Enabling Linux control group version 2 (cgroup v2)_. Updated intro text Added step to switch between v1 and v2. Added verification code blocks for v1.

Post-installation configuration -> Cluster tasks -> [Configuring Linux cgroup v2](https://56436--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/cluster-tasks.html#nodes-clusters-cgroups-2_post-install-cluster-tasks). Same module as above. Shows how to go from v1 to v2, as this is a post-install task.

Post-installation configuration -> Machine configuration tasks -> [Adding kernel arguments to nodes](https://56436--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/machine-configuration-tasks.html#nodes-nodes-kernel-arguments_post-install-machine-configuration-tasks)  Removed TP snippet after systemd.unified_cgroup_hierarchy 

OKD
Installing -> Installation configuration -> [Enabling Linux control group version 2](http://file.rdu.redhat.com/mburke/nodes-cgroup-v2-ga/installing/install_config/enabling-cgroup-v2.html)  

Nodes -> Working with clusters -> [Configuring the Linux cgroup on your nodes](http://file.rdu.redhat.com/mburke/nodes-cgroup-v2-ga/nodes/clusters/nodes-cluster-cgroups-okd.html) 

Post-installation configuration -> Cluster tasks -> [Configuring Linux cgroup v2](http://file.rdu.redhat.com/mburke/nodes-cgroup-v2-ga/post_installation_configuration/cluster-tasks.html#nodes-clusters-cgroups-okd-configure_post-install-cluster-tasks). 

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by mburke5678** [comment]
@sunilcio Can you PTAL and verify that there are no new features in the cgroup v2 GA release and that cgroup v1 is still the default in OCP? Thank you in advance!

File: modules/nodes-clusters-cgroups-2.adoc [hunk 1]
```diff
@@ -184,6 +226,27 @@ $ stat -c %T -f /sys/fs/cgroup
 ----
 cgroup2fs
 ----
+endif::post[]
+ifdef::nodes[]
+. Check that the `sys/fs/cgroup/cgroup2fs` or `sys/fs/cgroup/tmpfs` file is present on your nodes.
```
Comment (adellape) [inline modules/nodes-clusters-cgroups-2.adoc:231]: Colon on end.

File: post_installation_configuration/cluster-tasks.adoc [hunk 2]
```diff
@@ -630,10 +630,15 @@ include::modules/deploying-resource.adoc[leveloffset=+2]
 
 ifndef::openshift-origin[]
 include::modules/nodes-clusters-cgroups-2.adoc[leveloffset=+1]
+
+.Additional resources
```
Comment (adellape) [inline post_installation_configuration/cluster-tasks.adoc:634]: Add `[role="_additional-resources"]`

File: nodes/clusters/nodes-cluster-cgroups-2.adoc [hunk 3]
```diff
@@ -1,18 +1,15 @@
...
 
 toc::[]
 
-You can enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control group version 2] (cgroup v2) in your cluster by editing the `node.config` object. Enabling cgroup v2 in {product-title} disables all cgroups version 1 controllers and hierarchies in your cluster. cgroup v1 is enabled by default.
+By default, {product-title} uses  link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v1.html[Linux control group version 1] (cgroup v1) in your cluster. You can switch to link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control group version 2] (cgroup v2), if needed, by editing the `node.config` object. Enabling cgroup v2 in {product-title} disables all cgroup version 1 controllers and hierarchies in your cluster. 
 
 cgroup v2 is the next version of the Linux cgroup API. cgroup v2 offers several improvements over cgroup v1, including a unified hierarchy, safer sub-tree delegation, new features such as link:https://www.kernel.org/doc/html/latest/accounting/psi.html[Pressure Stall Information], and enhanced resource management and isolation. 
 
-:FeatureName: {product-title} cgroups version 2 support
```
Comment (adellape) [inline nodes/clusters/nodes-cluster-cgroups-2.adoc:13]: There's a remaining link to the FeatureGates section down in the "Additional resources" below in this assembly.

**Pull request merged**